sdk/nodejs/appservice/zMixins.ts

```diff
@@ -436,9 +437,9 @@ export interface ArchiveFunctionAppArgs extends FunctionAppArgsBase {
 };
 
 function createFunctionAppParts(
-        name: string,
-        args: ArchiveFunctionAppArgs,
-        opts: pulumi.CustomResourceOptions = {}) {
+    name: string,
+    args: ArchiveFunctionAppArgs,
+    opts: pulumi.CustomResourceOptions = {}) {

```

Comment:
i prefer the previous indentation because it makes it clear these are wrapped params, and not the starts of statements.